lambdas/process-journey-step/src/main/resources/statemachine/dev/ipv-core-main-journey.yaml

```diff
@@ -15,8 +15,9 @@ CRI_STATE:
       name: error
       targetState: CRI_ERROR
       response:
-        type: page
+        type: error
```

Comment:
I'd change this back for now until we stage it out with a feature flag

Reply:
Done this, and `@Disabled` tests for the time being.